monai/handlers/stats_handler.py

```diff
@@ -125,8 +134,8 @@ def _default_iteration_print(self, engine: Engine):
             engine (ignite.engine): Ignite Engine, it can be a trainer, validator or evaluator.
 
         """
-        prints_dict = engine.state.metrics
-        if not prints_dict:
+        loss = self.output_transform(engine.state.output)[1]
+        if loss is None:
```

Comment:
perhaps don't print it if state.output is high dimensional data?

Reply:
fixed in the latest patch.
Thanks.